src/components/Projects/WBS/WBSDetail/Task/Task.jsx

```diff
@@ -413,27 +413,11 @@ const Task = props => {
               <i className="fa fa-book" aria-hidden="true"></i>
             </td>
           </tr>
-
```

Comment:
Why was my feature deletion? Was this properly updated with the development branch before PR?